test/live_server.rb

```diff
@@ -1,4 +1,5 @@
 require 'sinatra/base'
+load 'test/helper.rb'
```

Comment:
Good point, if it is complaining is probably because that file gets loaded twice. Maybe changing `load` with `require` will fix the issue at requiring the same file twice only loads it once